packages/build/src/core/constants.ts

```diff
@@ -63,6 +63,10 @@ export interface NetlifyPluginConstants {
    * the Netlify site ID
    */
   SITE_ID: string
+  /**
+   * the Netlify account ID
+   */
+  ACCOUNT_ID: string
```

Comment:
Should this be nullable type, so plugins add some guards for cases of this not being set when older CLI is used?

I think it's nullable in this case
https://github.com/netlify/build/blob/14f2e9fef35817223c391fa2a9deb0bfc55ebf3c/packages/build/src/core/constants.ts#L31-L35

Reply:
Yep you're right, also the absence of the `--account-id` or if `--offline=false` might mean we won't have any.